lib/server.js

```diff
@@ -238,7 +243,7 @@ function handleResponse(opts, req, resp, response) {
         }
     } else {
         opts.log('error/request', {
-            req: req,
+            root_req: req,
```

Comment:
Yeah, good catch. However, I think that's actually not new in this patch. Perhaps we can address it in a follow-up?